stack/docker-compose.yml

```diff
@@ -0,0 +1,94 @@
+name: default-services
+
+services:
+  postgres:
+    container_name: postgres
+    image: postgres:16.0-alpine3.18
```

Comment:
Since Docker Compose 3.4 you can create reusable variables like this:

```
x-versions:
  postgres: &postgres_version 13.4
  redis: &redis_version 6.2.5
  app: &app_version 1.0.0
```

This would allow us to centralize versions of images.

To reuse variables you can use e.g:
```
image: postgres: *postgres_version
```

Reply:
one postgres ist explicit for the keycloak, the other is for local development. So if we bump the postgres version, both will get updated. If the keycloak version is not compatible with the postgres version there will be problems. And both postgresdbs have different use cases so either way i dont want to have any coupling between those two.

Reply:
Yeah that's fine. However I still suggest defining the image versions used at the top level of the docker-compose, for better maintainability.
We have the same concept in our pom.xml files

Reply:
Renovate is currently not capable of scanning the versions in a custom defined block (see https://github.com/nodeca/js-yaml?tab=readme-ov-file#caveats). Thats why need to drop this temporary.